config/config.go

```diff
@@ -40,9 +40,10 @@ const (
 	DefaultNodeKeyName  = "node_key.json"
 	DefaultAddrBookName = "addrbook.json"
 
-	MempoolTypeFlood = "flood"
-	MempoolTypeNop   = "nop"
-	MempoolTypeCAT   = "cat"
+	MempoolTypeFlood    = "flood"
+	MempoolTypeNop      = "nop"
+	MempoolTypePriority = "priority"
+	MempoolTypeCAT      = "cat"
```

Comment:
true, good catch @rootulp 

this was aparently a break in v0.34 which seems uneccesary https://github.com/celestiaorg/celestia-core/blob/23f7a57bec4a05d03685f53b6eb8a859ae115b3f/config/config.go#L27-L31

Reply:
can be addressed in follow up

Reply:
i can make it either or so it should work for either v1 or priority. Ill do it here. wait on the merge

Reply:
sg will do